libsmtutil/Sorts.h

```diff
@@ -194,6 +194,22 @@ struct SortProvider
 	static std::shared_ptr<IntSort> const uintSort;
 	static std::shared_ptr<IntSort> const sintSort;
 	static std::shared_ptr<IntSort> intSort(bool _signed = false);
+	static std::shared_ptr<BitVectorSort> const bitVectorSort;
+
+	static std::shared_ptr<Sort> sortFromKind(Kind _kind)
```

Comment:
@chriseth I'm not sure about this function. It works for `Int`, `Bool` and `BV`, but the other Sorts require more elements.